SQL console/src/com/serotonin/m2m2/db/dao/SqlConsole.java

```diff
@@ -50,13 +50,8 @@ public SqlQueryResult query(String sqlString, String serializedDataMsg) {
                         row.add(rs.getString(i + 1));
                     else if (meta.getColumnType(i + 1) == Types.LONGVARBINARY
                             || meta.getColumnType(i + 1) == Types.BLOB) {
-                        Blob blob = rs.getBlob(i + 1);
-                        Object o;
-                        if (blob == null)
-                            o = null;
-                        else
-                            o = SerializationHelper.readObjectInContext(blob.getBinaryStream());
-                        row.add(serializedDataMsg + "(" + o + ")");
+                        SerialBlob blob = new SerialBlob( rs.getBlob(i + 1) );
```

Comment:
@ManuelCalvo3Pillar you have removed the ability for Mango to display the content of the fields that are manually serialized by Mango.  Take a look at how that works by looking at the data column on the `dataPoints` , `dataSources` or `publishers` tables.  

If you can find a solution that retains that ability and fixes this bug then I will approve it.

Reply:
hi @terrypacker , let me see if I understand: you are saving serialized object on this Blob fields right? Are we going to support the user making queries to Blob fields that don't have object in it? If that is true, the bytes can be anything and can be quite large to display is it OK if I just you string saying something like "BLOB" as the value for that column? So the logic if lake this: if it can parse it as a object show the toString method of the object, if not return the string "BLOB". Would that be OK?

Reply:
What I'm saying is that we already support displaying the contents.  So you need to keep that functionality when you make any new changes.

![image](https://user-images.githubusercontent.com/3905204/221728568-3c06811a-3709-4886-ac38-3ae42d444590.png)

Reply:
So to handle the case of a non-Mango serialized blob you can try/catch the existing SerializationHelper code and then use your code.  This way we keep the ability to de-serialize the mango objects still.  We never thought someone would want to use the SQL console to explore rows with non Mango blobs in them.

Reply:
I implemented the change as discussed.